src/Types.elm

```diff
@@ -0,0 +1,13 @@
+module Types exposing (..)
+
+type Note
+    = C
+    | DSharp
+    | FSharp
+    | A
+
+type alias NoteAlias = String
```

Comment:
Why not just use 'Note' instead of making 'NoteAlias'?

Reply:
The types of values that can flow through in and out of Elm include:

        Ints, Floats, Bools, Strings, Maybes, Lists, Arrays, Tuples, Json.Values,
        and concrete records.

This has to be a String. I defined the alias to make it more expressive.